examples/example-multiplot.cpp

```diff
@@ -47,8 +47,8 @@ int main(int argc, char** argv)
     plot3.drawCurve(x, std::sqrt(x)).label("sqrt(x)");
 
     // Use the previous plots as sub-figures in a larger 2x2 figure.
-    Figure fig = {{ plot0, plot1 },
-                  { plot2, plot3 }};
+    Figure fig = {{ &plot0, &plot1 },
+                  { &plot2, &plot3 }};
```

Comment:
This is dangerous because there will be certainly a user in the future creating a function that returns a `Figure` object with the `Plot` objects created in the function scope (which gets deleted at the end of said scope). These points will then be dangling. 

I would say we can work with copies. If this becomes problematic one day, we wrap the entire data inside a `Plot` in a `std::shared_ptr` so that we can work safely with references.

Reply:
You are right, that is why the commit is called "Probably not the best idea , .."

Reply:
Raw pointers are so last century ;)

Reply:
Plus if plot0-3 go out of scope this will crash